hieradata/common/common.yaml

```diff
@@ -229,8 +232,9 @@ profile::network::services::dns_records:
     "logger.%{hiera('domain_mgmt')}":                       "%{hiera('netcfg_mgmt_netpart')}.13"
     "monitor.%{hiera('domain_mgmt')}":                      "%{hiera('netcfg_mgmt_netpart')}.14"
     "builder.%{hiera('domain_mgmt')}":                      "%{hiera('netcfg_mgmt_netpart')}.15"
-    "ns01.%{hiera('domain_mgmt')}":                         "%{hiera('netcfg_mgmt_netpart')}.16"
-    "ns02.%{hiera('domain_mgmt')}":                         "%{hiera('netcfg_mgmt_netpart')}.17"
+    "ns.%{hiera('domain_mgmt')}":                           "%{hiera('netcfg_mgmt_netpart')}.16"
```

Comment:
Skal DNS-tjestene på disse tilbys på mgmt via domenenavn? Ellers kan vi ta det vekk. De andre som ligger der er webtjenester (unntatt git) som faktisk skal tilbys på mgmt med et domenenavn som er forskjellig fra fqdn.
(Dette gjelder alle A-records for $domain_mgmt som IKKE er FQDN)

Reply:
Det kan tas vekk